recipes/openimageio/all/conanfile.py

```diff
@@ -169,78 +177,85 @@ def generate(self):
+
+        # Not supported yet with the conan package.
         tc.variables["USE_R3DSDK"] = False
         tc.variables["USE_NUKE"] = False
         tc.variables["USE_OPENGL"] = False
         tc.variables["USE_QT"] = False
-        tc.variables["USE_LIBPNG"] = self.options.with_libpng
-        tc.variables["USE_FREETYPE"] = self.options.with_freetype
-        tc.variables["USE_LIBWEBP"] = self.options.with_libwebp
-        tc.variables["USE_OPENJPEG"] = self.options.with_openjpeg
 
-        if self.options.with_ffmpeg:
-            tc.cache_variables["USE_FFMPEG"] = self.options.with_ffmpeg
-            tc.cache_variables["CMAKE_REQUIRE_FIND_PACKAGE_FFmpeg"] = True
-            tc.cache_variables["FFMPEG_VERSION"] = f'"{str(self.dependencies["ffmpeg"].ref.version)}"'
+        # Dependencies that are disabled will not call find_package, so to make sure that all reqeusted
+        # modules are integrated, we can just force require to ensure if not disabled via USE_*, missing
+        # find_package calls are silently failing.
+        tc.cache_variables["CMAKE_REQUIRE_FIND_PACKAGE_libjpeg-turbo"] = self.options.with_libjpeg != "libjpeg-turbo"
+        tc.cache_variables["CMAKE_REQUIRE_FIND_PACKAGE_JPEG"] = True  # This will only be evaluated if libjpeg-turbo failed
+        tc.cache_variables["CMAKE_REQUIRE_FIND_PACKAGE_JXL"] = True
+        tc.cache_variables["CMAKE_REQUIRE_FIND_PACKAGE_PNG"] = True
+        tc.cache_variables["CMAKE_REQUIRE_FIND_PACKAGE_Freetype"] = True
+        tc.cache_variables["CMAKE_REQUIRE_FIND_PACKAGE_OpenColorIO"] = True
+        tc.cache_variables["CMAKE_REQUIRE_FIND_PACKAGE_OpenCV"] = True
+        tc.cache_variables["CMAKE_REQUIRE_FIND_PACKAGE_TBB"] = True
+        tc.cache_variables["CMAKE_REQUIRE_FIND_PACKAGE_DCMTK"] = True
+        tc.cache_variables["CMAKE_REQUIRE_FIND_PACKAGE_FFmpeg"] = True
+        tc.cache_variables["CMAKE_REQUIRE_FIND_PACKAGE_GIF"] = True
+        tc.cache_variables["CMAKE_REQUIRE_FIND_PACKAGE_Libheif"] = True
+        tc.cache_variables["CMAKE_REQUIRE_FIND_PACKAGE_LibRaw"] = True
+        tc.cache_variables["CMAKE_REQUIRE_FIND_PACKAGE_OpenJPEG"] = True
+        tc.cache_variables["CMAKE_REQUIRE_FIND_PACKAGE_openjph"] = True
+        tc.cache_variables["CMAKE_REQUIRE_FIND_PACKAGE_OpenVDB"] = True
+        tc.cache_variables["CMAKE_REQUIRE_FIND_PACKAGE_Ptex"] = True
+        tc.cache_variables["CMAKE_REQUIRE_FIND_PACKAGE_WebP"] = True
+        tc.cache_variables["CMAKE_REQUIRE_FIND_PACKAGE_libuhdr"] = True
 
-        if Version(self.version) < "3.0":
-            tc.cache_variables["Boost_USE_STATIC_LIBS"] = not self.dependencies["boost"].options.shared
-        tc.cache_variables["BUILD_MISSING_ROBINMAP"] = False
+        # Prepare some internal
         tc.cache_variables["CMAKE_REQUIRE_FIND_PACKAGE_Robinmap"] = True
         tc.cache_variables["CMAKE_REQUIRE_FIND_PACKAGE_pugixml"] = True
-        tc.cache_variables["INTERNALIZE_FMT"] = False
```

Comment:
fairly certain still needed, but if it is not, please add comments to the PR with how you've arrived to that conclusion